i18n/es.json

```diff
@@ -6511,19 +6511,19 @@
     "translation": "No se puede actualizar la fecha de la última actualización del icono del equipo"
   },
   {
-    "id": "store.sql_terms_of_service.save.app_error",
+    "id": "app.terms_of_service.create.app_error",
```

Comment:
The expectation here is to only update the `en.json` file

Reply:
Ok @jespino I will remove all changes in other language files but, what will happen with the thoses file containing outdated keys ? Because here I'm not adding new ones, but modifying.